content/en/status.md

```diff
@@ -37,6 +34,14 @@ The development status, or maturity level, of the [per-language SDKs](/docs/inst
 
 </div>
 
+The development status, or maturity level, of the [collector](/docs/collector/) and [specification](/docs/reference/specification/) are as follows:
```

Comment:
Can we replace this as well with a similar wording as above:

> For the development status, or maturity level, of the [collector](/docs/collector/) and [specification](/dosc/reference/specification/), see the status section pages:

Reply:
Hmm, that wording doesn't quite work (the collector has a status _section_, but the spec doesn't -- the entire page is dedicated to its status). How about:

> For the development status, or maturity level, of the [collector](/docs/collector/) and [specification](/docs/reference/specification/), see the following:

Reply:
@chalin I went with your suggestion.